templates/emails/ticket.confirmed.hbs

```diff
@@ -26,9 +26,7 @@ Subject: {{ order.quantity }} {{pluralize "ticket" n=order.quantity}} confirmed
 
 {{#if event.data.privateInstructions}}
 <h2>Private Instructions</h2>
-<p>
-  {{event.data.privateInstructions}}
-</p>
+<pre>{{event.data.privateInstructions}}</pre>
```

Comment:
The styling is a bit rough. Can we try re-using something from another email? For example `templates/emails/collective.comment.created.hbs`:

```suggestion
<table style="width: 100%; margin-bottom: 30px;">
  <tbody>
    <tr>
      <td style="border: 1px solid #e8edee; border-radius: 6px; padding: 2em 1em;">
        <blockquote style="color: #798088;font-size: 13px;text-align: left;padding: 0px 1em;margin: 0em 0;white-space: pre-line;">{{event.data.privateInstructions}}</blockquote>
      </td>
    </tr>
  </tbody>
</table>
```

Could you try something along those lines, eventually play with the styles if there are issues, and leave a screenshot? Thanks!

Reply:
@Betree This is how it looks now, and I feel it's looking good now.
<img width="699" alt="Screenshot 2022-07-09 at 3 54 00 PM" src="https://user-images.githubusercontent.com/46647141/178101724-4c2ad062-d4c8-498d-a39b-9501c8ac7056.png">